### PR TITLE
some edits on dss example

### PR DESCRIPTION
**Pull request by agramfort**
just some random edits on DSS example while discovering it.

btw I realize that the API is not settle there. It should work like the ICA object with 

```
dss = DSS(data_thresh=1e-9, bias_thresh=1e-9, n_components)
dss.fit(epochs)
epochs_clean = dss.apply(epochs)
evoked_clean = dss.apply(epochs.average())
```

### CODE REVIEW

**Comment by Eric89GXL** [comment]
@agramfort do you want to change the API? Maybe we could do it if we decide to move it to `mne`.

@drammock can you have a look at the changes?

**Comment by agramfort** [comment]
what would be great would be use use DSS or ICA by just changing one line
of code like a sklearn estimator that just has a fit and apply method in
our case.

File: examples/preprocessing/plot_dss_epochs.py [hunk 1]
```diff
@@ -6,44 +6,54 @@
...
+
 # plot
-fig, axs = plt.subplots(2, 1, figsize=(7, 7), sharex=True)
-plotdata = [evoked.data.T, dss_data[:, 0].T]
-linewidths = (1, 0.6)
+fig, axs = plt.subplots(3, 1, figsize=(7, 10), sharex=True)
+plotdata = [evoked.data.T, evoked_data_clean.T, dss_data[:, 0].T]
+linewidths = (1, 1, 0.6)
 titles = ('evoked data (EEG only)',
+          'evoked data (clean)',
```
Comment (drammock) [inline examples/preprocessing/plot_dss_epochs.py]: I think this would be clearer as `evoked data after DSS (EEG only)`

**Comment by drammock** [comment]
other than clarifying the new subplot's title LGTM.  I definitely won't have time to do the API adjustment for the next 2 weeks at least, but am willing to give it a shot later on if nobody else picks it up.

**Comment by agramfort** [comment]
done

**Pull request merged**